src/components/PDiskInfo/PDiskInfo.tsx

```diff
@@ -107,14 +169,49 @@ export function PDiskInfo<T extends PreparedPDisk>({
                             external={false}
                         />
                     )}
-                    <LinkWithIcon
-                        title={pDiskInfoKeyset('developer-ui')}
-                        url={pDiskInternalViewerPath}
-                    />
+                    {isUserAllowedToMakeChanges && (
+                        <LinkWithIcon
+                            title={pDiskInfoKeyset('developer-ui')}
+                            url={pDiskInternalViewerPath}
+                        />
+                    )}
                 </span>
             ),
         });
     }
 
-    return <InfoViewer info={pdiskInfo} {...infoViewerProps} />;
+    return [generalInfo, statusInfo, spaceInfo, additionalInfo];
+}
+
+interface PDiskInfoProps<T extends PreparedPDisk> extends GetPDiskInfoOptions<T> {
+    className?: string;
+}
+
+export function PDiskInfo<T extends PreparedPDisk>({
+    pDisk,
+    nodeId,
+    isPDiskPage = false,
+    className,
+}: PDiskInfoProps<T>) {
+    const {isUserAllowedToMakeChanges} = useTypedSelector((state) => state.authentication);
+
+    const [generalInfo, statusInfo, spaceInfo, additionalInfo] = getPDiskInfo({
+        pDisk,
+        nodeId,
+        isPDiskPage,
+        isUserAllowedToMakeChanges,
+    });
+
+    return (
+        <div className={b('wrapper', className)}>
+            <div className={b('col')}>
+                <InfoViewer info={generalInfo} renderEmptyState={() => null} />
```

Comment:
What do you think if we start using `DefinitionList` from `@gravity-ui/components`?

Reply:
I try to use it for new components (e.g. `ViewInfo`), but don't refactor old components (maybe someday I'll do it separately, currently I don't want to mix features with refactoring)